ddp-workspace/projects/ddp-dsm-ui/src/app/field-datepicker/field-datepicker.component.html

```diff
@@ -29,6 +29,8 @@
           *ngIf="showTodayButton"
           (click)="setToday()"
           [disabled]="disabled">Today</button>
+  <button *ngIf="showSaveBtn" class="btn btn-xs Button--Background--Color ng-star-inserted"
```

Comment:
This button is still clickable even during the saving process (while the request is in the pending state), and later as well, whether it fails or not, which, I think, is not quite a user-friendly approach. What do you think?

Reply:
I think it is fine if it is clickable after failing, because the user may want to try again. However, I can disable it during the pending state.